spec/rails_helper.rb

```diff
@@ -2,7 +2,9 @@
 
 # This file is copied to spec/ when you run 'rails generate rspec:install'
 require 'spec_helper'
-require 'factories/factory_bot.rb'
+require 'factories/books.rb'
+require 'factories/users.rb'
+require 'factories/rents.rb'
```

Comment:
mmm... I think this it's not scalable. What do you think?

Reply:
not at all, sry my bad, solved